packages/fileeditor/src/tabspacestatus.tsx

```diff
@@ -49,7 +49,13 @@ function TabSpaceComponent(
       : trans.__('Tab Indent');
   return (
     <TextItem
+      role="button"
```

Comment:
Same as with others:

```suggestion
      role="button"
      aria-haspopup="true"
      tabIndex={0}
```

At this point I wonder if we should have a wrapper around `TextItem` say `PopupButtonItem` as this is a frequent use case. And ideally it should also have `aria-controls` pointing to the popup.

Reply:
@krassowski I do not see many items having popups to add a new wrapper class. We could just pass in the props as needed from the caller. 
When testing I noticed that the DOM order is reversed for the "right" section of the footer toolbar. So the tab focus will go in reverse order(shown in video attached). Also, I added `tabIndex` to all elements as they contain some data that could be announced by the screen readers. Do you have any thoughts on this? Should we only allow tabbing to elements that have an action? 

https://github.com/user-attachments/assets/76f6cb8c-6278-426d-a1d0-f347d37742a4